packages/angular_devkit/build_angular/src/tslint/schema.json

```diff
@@ -43,16 +44,24 @@
       "type": "string",
       "description": "Output format (prose, json, stylish, verbose, pmd, msbuild, checkstyle, vso, fileslist).",
       "default": "prose",
-      "enum": [
-        "prose",
-        "json",
-        "stylish",
-        "verbose",
-        "pmd",
-        "msbuild",
-        "checkstyle",
-        "vso",
-        "fileslist"
+      "anyOf": [
```

Comment:
What's the difference between just having the `enum` and having `anyOf` with the `enum` and `minLength`?

Reply:
The second allows specifying custom formatters.  TSLint will try to load a formatter if it is not one of the builtin options.